drivers/gpio/Kconfig.pca_series

```diff
@@ -0,0 +1,50 @@
+# PCA_SERIES GPIO configuration options
+
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig GPIO_PCA_SERIES
+	bool "PCA_SERIES I2C-based GPIO chip"
+	default y
+	depends on DT_HAS_NXP_PCA9538_ENABLED || DT_HAS_NXP_PCA9539_ENABLED || \
+		   DT_HAS_NXP_PCA9574_ENABLED || DT_HAS_NXP_PCA9575_ENABLED || \
+		   DT_HAS_NXP_PCAL9538_ENABLED || DT_HAS_NXP_PCAL9539_ENABLED || \
+		   DT_HAS_NXP_PCAL6524_ENABLED || DT_HAS_NXP_PCAL6534_ENABLED
+	depends on I2C
+	help
+	  Enable driver for PCA_SERIES I2C-based GPIO chip.
+
+# TODO: add this line
+#		   DT_HAS_NXP_PCAL6408A_ENABLED || DT_HAS_NXP_PCAL6416A_ENABLED || \
+
+config GPIO_PCA_SERIES_INIT_PRIORITY
+	int "Init priority"
+	default 70
+	depends on GPIO_PCA_SERIES
+	help
+	  Device driver initialization priority.
+
+config GPIO_PCA_SERIES_INTERRUPT
+	bool "Interrupt enable"
+	depends on GPIO_PCA_SERIES
+	help
+	  Enable interrupt support in PCA_SERIES driver.
+	  Note that the PCA_SERIES cannot reliably detect
+	  short-pulse interrupts due to its design.
+
+config GPIO_PCA_SERIES_CACHE_ALL
+	bool "Cache all registers"
+	depends on GPIO_PCA_SERIES
+	help
+	  Cache all registers in RAM for faster configuration.
+	  Enabled by default.
+	  When disabled, only output registers will be cached.
+	  On devices w/o interrupt status, it will also cache
+	  input state.
+
+config GPIO_PCA_SERIES_REGISTER_DUMP
```

Comment:
Sorry that my prior comment wasn't clear- we can drop this Kconfig, or we can leave it in and make the code for register dumps depend on `CONFIG_GPIO_PCA_SERIES_REGISTER_DUMP`. Currently it depends on `GPIO_PCA_SERIES_REGISTER_DUMP`. Either method is ok with me, I just think we should be consistent

Reply:
Removed. Now `GPIO_NXP_PCA_SERIES_DEBUG` controls both register dump and extra checking.